### ISSUE TITLE
Display article type 

### ISSUE THREAD

**Issue by Maelplaine** [libero/libero#6]
## Problem / Motivation
- WHO reader
- WHEN landing on the article page
- WHERE next to the publication date in the article header, below the authors

## Proposed solution
- Articles have a type

## Clarification needed
- Is the generic word “Article type” fine? What are the other journals using?
> IJM is using “content type” but is happy to use "article type"
- Are we doing anything with the article type other than displaying it alongside the title?
> Yes we only display the article type. It will become a link once the search milestone is done 
- Do we need different schemas per type? (eg eLife’s scholarly articles vs blog articles)
> IJM doesn’t need different schemas. See different types below:
- List: research-article - Research article; article-commentary - Commentary brief-report - Research note editorial - Editorial product-review - Data watch, Software review book-review - Book review correction - Correction

## Assumptions
- No defined vocabulary.

## Tasks
- [x] Add to content header pattern
- [ ] Use first `{http://jats.nlm.nih.gov}subject` from `{http://jats.nlm.nih.gov}sub-group@subj-group-type="display-channel"`

## Technical notes
- We won't display what JATS calls `article-type` the right value to use is the content in the `subject` tag within the `subj-group` with the attribute `subj-group-type="display-channel"`

## User interface / Wireframes

![image](https://user-images.githubusercontent.com/39725642/43400420-8e0ebd4a-9405-11e8-9521-c2f2837b55b4.png)

**Comment by discodavey** [comment]
https://github.com/libero/pattern-library/pull/57

**Comment by discodavey** [comment]
Putting this on hold until @chugginselifesciences gets back off holiday as there are some questions on design.

**Comment by discodavey** [comment]
This is being merged into item-tags pattern so is now on a different PR.

https://github.com/libero/pattern-library/pull/64

**Comment by discodavey** [comment]
After speaking to @chugginselifesciences it was agreed that these would always be links

**Comment by discodavey** [comment]
Closed PR 64 as reverting back to previous PR

https://github.com/libero/pattern-library/pull/57

**Comment by thewilkybarkid** [comment]
Browser PR: https://github.com/libero/browser/pull/55

**Comment by thewilkybarkid** [comment]
JATS support: https://github.com/libero/jats-support/pull/9
